src/components/VideoCapture/index.tsx

```diff
@@ -226,6 +228,7 @@ export default class VideoCapture extends Component<Props, State> {
 
     return (
       <Camera
+        idealCameraWidth={IDEAL_CAMERA_WIDTH_IN_PX}
```

Comment:
I believe we have some bugs in different Android devices, just didn't remember what. @stefaniacardenas do you remember any about this?

Reply:
From `src/components/Camera/index.tsx`
```
// Specify just a camera width (no height) because on safari if you specify both
  // height and width you will hit an OverconstrainedError if the camera does not
  // support the precise resolution.
  // Using width here because on some special devices (e.g. Samsung Galaxy),
  // setting 720px in height results to 720x960 resolution instead of the desired 720x1280.
  //
  // Resolution needs to be set to a lower value if WebM format is not supported
  // as video formats like MP4 (Safari) result in larger video file sizes
  // * Resolutions: 1280px = 720p, 480px = VGA (VGA is minimum we can go for automation + iOS SDK is using VGA resolution)
  
  Plus https://github.com/onfido/react-webcam/blob/master/src/react-webcam.js#L200-L211